test/calcium/qqbar-test.jl

```diff
@@ -35,38 +35,39 @@
 end
 
 @testset "QQBarFieldElem.printing" begin
-   a = CalciumQQBar(1)
+   QQBar = algebraic_closure(QQ)
+   a = QQBar(1)
 
    @test string(a) == "Root 1.00000 of x - 1"
    @test string(parent(a)) == "Field of algebraic numbers"
 
    @test string(-(QQBarFieldElem(10) ^ 20)) == "Root -1.00000e+20 of x + 100000000000000000000"
-   @test string(root_of_unity(CalciumQQBar, 3)) == "Root -0.500000 + 0.866025*im of x^2 + x + 1"
+   @test string(root_of_unity(QQBar, 3)) == "Root -0.500000 + 0.866025*im of x^2 + x + 1"
    @test string(sqrt(QQBarFieldElem(-1)) // 3) == "Root 0.333333*im of 9x^2 + 1"
 end
 
 
 @testset "QQBarFieldElem.manipulation" begin
-   R = CalciumQQBar
-
-   @test zero(R) == 0
-   @test one(R) == 1
-   @test isa(zero(R), QQBarFieldElem)
-   @test isa(one(R), QQBarFieldElem)
-   @test zero(R) == zero(QQBarFieldElem)
-   @test one(R) == one(QQBarFieldElem)
-
-   @test iszero(R(0))
-   @test isone(R(1))
-   @test is_rational(R(1))
-   @test isreal(R(1))
-   @test degree(R(1)) == 1
-
-   u = sqrt(R(2))
-   i = sqrt(R(-1))
+   QQBar = algebraic_closure(QQ)
```

Comment:
If this was just
```suggestion
   R = algebraic_closure(QQ)
```
the diff would be much smaller